dfetch/__main__.py

```diff
@@ -42,6 +43,7 @@ def create_parser() -> argparse.ArgumentParser:
     dfetch.commands.freeze.Freeze.create_menu(subparsers)
     dfetch.commands.update.Update.create_menu(subparsers)
     dfetch.commands.validate.Validate.create_menu(subparsers)
+    dfetch.commands.list.List.create_menu(subparsers)
```

Comment:
Very unimportant, but could you sort the commands the same way they are sorted with the imports?